### DISCUSSION TITLE
32 -2, 32 - 3, 33 - 2 문제 이해를 못하겠습니다!

### DISCUSSION THREAD

**Discussion by Mang-Joo** [I-JUNNYLAND-I/algorithm#166] [Q&A] [answered]
32 - 2
32 - 3
33 - 2 문제 이해를 못했습니다. ㅠㅠ

**Comment by Junnyjun** [comment, marked as answer]
32 -2 )
배열중 원소의 개수가 적지만 합이 최대가 되게끔 나누는 문제로 인식후 풀었습니다.
A,B,C,D,E (order)
A~D 합 < E 합 우선순위를 생각하며 풀었습니다.

4,3,8,9,10 => 
4,3,8 (합 15) / 9,10 (합 19)

4,4,7,6,7 =>
4,4,6 (14) /7,7 (14) 동일하니 X
4,4 (8) / 6,7,7 (20) 최소원소&합최대

**Comment by Junnyjun** [comment]
32-3은 저도모르겟어요 ㅎ,ㅎ

**Comment by Junnyjun** [comment]
33-2 
꽃(1)은 화단(0)에 한칸씩 띄워 심어야 하는데.

1,0,0,0,0,0,1 이면 가운데 1,0,X,0,X,0,1 심을수 잇는 꽃 (X)의 개수가 주어진 수 보다 작은지 확인하는 문제입니다.
